src/qt/editor/inviwomainwindow.cpp

```diff
@@ -130,7 +130,14 @@ InviwoMainWindow::InviwoMainWindow(InviwoApplicationQt* app)
                              "path"}
     , openData_{"d", "data", "Try and open a data file", false, "", "file name"}
     , updateWorkspaces_{"", "update-workspaces",
-                        "Go through and update all workspaces to the latest version"}
+                        "Update workspaces of all modules to the latest version "
+                        "(located in '<module>/data/workspaces/*')"}
```

Comment:
module->modules

Reply:
don't agree. This refers to the respective module path. You maybe refer to `modules/<module>/data/...`?

Reply:
Yes, you are correct